applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -38,7 +38,7 @@ LOG_MODULE_REGISTER(at_host, CONFIG_SLM_LOG_LEVEL);
 #define FATAL_STR	"FATAL ERROR\r\n"
 #define SLM_SYNC_STR	"Ready\r\n"
 
-#define SLM_VERSION	"#XSLMVER: 1.2\r\n"
+#define SLM_VERSION	"#XSLMVER: 1.4\r\n"
 #define AT_CMD_SLMVER	"AT#XSLMVER"
```

Comment:
What is the rule for the SLM versioning?

Reply:
We don't really have a rule. Let's follow the NCS version. Current SLM is for NCSv1.4.